internal/services/eventhub/eventhub_resource.go

```diff
@@ -163,6 +162,36 @@ func resourceEventHub() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOhBeta() {
+		r.Schema["namespace_id"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ExactlyOneOf: []string{"namespace_id", "namespace_name"},
+			ValidateFunc: namespaces.ValidateNamespaceID,
+		}
+
+		r.Schema["namespace_name"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validate.ValidateEventHubNamespaceName(),
+			ExactlyOneOf: []string{"namespace_id", "namespace_name"},
+			Deprecated:   "`namespace_name` has been deprecated in favour of `namespace_id`",
```

Comment:
Can't believe I didn't catch this in the previous review, but could we update this so it's consistent with other deprecation messages in the provider and also clear in that it will be removed in the next major release 
```suggestion
			Deprecated:   "`namespace_name` has been deprecated in favour of `namespace_id` and will be removed in v5.0 of the AzureRM Provider",
```